src/wmnds/components/content-tiles/map/map.njk

```diff
@@ -0,0 +1,16 @@
+{% macro wmndsContentTileMap(params) %}
+
+<div class="wmnds-content-tile">
+    <div class="wmnds-content-tile__title">
+        <h2>Driving from Bearwood</h2>
+    </div>
+    <div class="wmnds-content-tile--copy">
+        <p>Consider using City Road and Dudley Road, past City Hospital to head into Birmingham city centre or connect with the ring road.</p>
+    </div>
+    <div class="map">
+        <img src="http://placehold.it/600x400" alt="empty image" class="wmnds-col-1">
```

Comment:
Just make the alt a bit more meaningful i.e. "map of the west midlands".

Also, can you remove the class "wmnds-col-1" as grid elements shouldn't be used within a component.